wasm/src/api.rs

```diff
@@ -40,3 +53,82 @@ pub fn run_clvm(program: &[u8], args: &[u8]) -> Vec<u8> {
         Err(_eval_err) => format!("{:?}", _eval_err).into(),
     }
 }
+
+#[wasm_bindgen]
+/**
+ * Return serialized result of clvm program with cost
+ *
+ * cost will be available at the first 8 bytes of returned Vec<u8>(=Uint8Array).
+ * bytes of node will be available from 8th byte offset of returned Vec<u8>.
+ *
+ * @example
+ * const result = run_chia_program(...); // Uint8Array
+ * const cost = vec_u8_to_u64(result.subarray(0, 8)) // BigInt;
+ * const serialized_node = result.subarray(8); // Uint8Array
```

Comment:
this sounds a bit awkward. you can't return tuples back to javascript?

If you end up concatenating, you may want to be careful with how you do it. Right now you end up copying `node_bytes` into a new buffer. `nodes_bytes` is potentially quite large. You could for instance insert the 8 bytes of cost at the beginning instead. It might end up copying still, but it might also `memmov()` the bytes if there's enough capacity in the vector.

Reply:
There are no built-in `tuple` type in JavaScript so we need to use JavaScript `Array` as a tuple.
Creating and returning JavaScript's `Array` in Rust/Wasm code seems heavy.
Reference: https://rustwasm.github.io/docs/wasm-bindgen/reference/arbitrary-data-with-serde.html

If we want to return a tuple like `[cost, node_bytes]`, the type of returning tuple would be `[u64, vec<u8>]` and the composite type like this seems requiring JSON serialization/deserialization with `serde` rust package.
So instead of doing such a slow task, I preferred a bit painful but performant way to get cost/node_bytes.

Reply:
I agree that JSON would probably not carry its weight here.